tests/test_arguments.py

```diff
@@ -0,0 +1,12 @@
+import rasa_sdk.endpoint as ep
+
+
+def test_arg_parser_endpoints():
```

Comment:
Final comment: can we parametrize this test so that it checks for default in one test, and for the specified param in another case?